Makefile

```diff
@@ -45,6 +45,18 @@ install: go.sum
 
 .PHONY: build install
 
+###############################################################################
+##                                  Docker                                   ##
+###############################################################################
+
+docker-build:
+	@docker build -t ghcr.io/ojo-network/price-feeder-ojo .
```

Comment:
```suggestion
	@DOCKER_BUILDKIT=1 docker build -t ghcr.io/ojo-network/price-feeder-ojo .
```

Reply:
https://github.com/ojo-network/price-feeder/pull/62/commits/dfd10021c388523f55cc310477f9b5eb1e34cb41

Reply:
you could just accept the suggestion ;)